modules/guides/pages/select.adoc

```diff
@@ -119,6 +119,8 @@ The set of fields returned by the query is known as the _projection_.
 NOTE: Before you can query a data source, there must be an index on that data source.
 The sample dataset already contains all the indexes you need to run the example queries in this guide.
 
+NOTE: We use backticks around the travel-sample dataset because its name contains a hyphen.
```

Comment:
I would put this in the example itself, i.e. after the query but before the result. Also I would make it just a line of text, not a note, just trying to keep visual noise and distraction to a minimum.